semantic_conventions/resource/cloud.yaml

```
@@ -47,6 +47,32 @@ groups:
           [Google Cloud regions](https://cloud.google.com/about/locations),
           or [Tencent Cloud regions](https://intl.cloud.tencent.com/document/product/213/6091).
         examples: ['us-central1', 'us-east-1']
+      - id: resource_id
+        type: string
+        brief: >
+          A unique identifier for a specific cloud resource.
```

I think we need to be more specific here:

```suggestion
          Cloud provider-specific native identifier of the monitored cloud resource (e.g. an [ARN](https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html) on AWS, an [fully qualified resource ID](https://learn.microsoft.com/en-us/rest/api/resources/resources/get-by-id) on Azure, a [full resource name](https://cloud.google.com/apis/design/resource_names#full_resource_name) on GCP)
```